### PR TITLE
Windows related tweaks for the build environment

### PR DESCRIPTION
**Pull request by benoit-pierre**
- update build instructions and batch file
- update setup instructions
- add batch file to setup environment

### CODE REVIEW

File: windows/README.md [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
+- `pip install -r requirements.txt`
+
+### Running Plover in development
 
 To run from source, from the root of the Git repository, use `python launch.py`.
 
 ### Building
 
-As a prerequesite, ensure that there is a file named `__init__.py` in the pywinusb directory in site-packages. If not, create it (as an empty file)
+As a pre-requisite, ensure that there is a file named `__init__.py` in the pywinusb directory in site-packages. If not, create it (as an empty file).
```
Comment (benoit-pierre) [inline windows/README.md]: Is it still required? @morinted: does your Treal work with the AppVeyor builds?

Comment (morinted) [inline windows/README.md]: As far as I'm aware, pip-installed pywinusb never had that problem.

Comment (benoit-pierre) [inline windows/README.md]: Okay, so is there any point in keeping it?

Comment (morinted) [inline windows/README.md]: No, we should remove that detail.

Comment (benoit-pierre) [inline windows/README.md]: OK, removed.

**Pull request merged**